web/client/components/map/openlayers/plugins/WMSLayer.js

```diff
@@ -29,16 +31,63 @@ function getWMSURLs( urls ) {
     return urls.map((url) => url.split("\?")[0]);
 }
 
+function needProxy(uri) {
```

Comment:
we should extenalize in a utils function. axios already have a function like this, isn't it?